go-controller/pkg/util/nicstobridge.go

```diff
@@ -148,8 +156,48 @@ func saveRoute(oldLink, newLink netlink.Link, routes []netlink.Route) error {
 	return nil
 }
 
+func runningPlatform(fs afero.Fs) (string, error) {
```

Comment:
i dont really think this should be in this file, although I do think it should be in some util/ file

Reply:
This is the only place it's used. If somewhere else used it, sure.